index.js

```diff
@@ -36,7 +36,7 @@ function HTMLReactParser(html, options) {
 HTMLReactParser.domToReact = domToReact;
 HTMLReactParser.htmlToDOM = htmlToDOM;
 HTMLReactParser.attributesToProps = attributesToProps;
-HTMLReactParser.Element = require('domhandler/lib/node').Element;
+HTMLReactParser.Element = require('domhandler').Element;
```

Comment:
This change increases bundle size, which causes Size Limit to fail.

Do you mind running `size-limit` and updating `.size-limit.json`?

```sh
npm run size-limit
```

Reply:
done

Reply:
> This change increases bundle size, which causes Size Limit to fail.
> 
> Do you mind running `size-limit` and updating `.size-limit.json`?
> 
> ```shell
> npm run size-limit
> ```

done

Reply:
Thanks!